src/CacheInvalidator.php

```diff
@@ -279,12 +302,11 @@ public function invalidateRegex($path, $contentType = null, $hosts = null)
      */
     public function invalidateTags(array $tags)
     {
-        if (!$this->cache instanceof BanInterface) {
+        if (!$this->tagHandler) {
             throw UnsupportedProxyOperationException::cacheDoesNotImplement('BAN');
         }
 
-        $headers = array($this->getTagsHeader() => '('.implode('|', array_map('preg_quote', $tags)).')(,.+)?$');
-        $this->cache->ban($headers);
+        $this->tagHandler->invalidateTags($tags);
```

Comment:
With only such a simple proxy the value of the `CacheInvalidator` facade becomes less clear, so we may want to remove `CacheInvalidator::invalidateTags()` later. Still, let’s keep the `invalidateTags()` method for BC.